### ISSUE TITLE
bug(Field.Selection): keyboard navigation in variant radio

### ISSUE THREAD

**Issue by langz** [dnbexperience/eufemia#3840]
## 🐛 Bug Report

The keyboard navigation in [Field.Selection in variant radio](https://eufemia.dnb.no/uilib/extensions/forms/base-fields/Selection/demos/#radio-buttons) does not work as expected(compared to [Eufemia's Radio component](https://eufemia.dnb.no/uilib/components/radio/#radio-group)).
In [Field.Selection with variant radio](http://localhost:8000/uilib/extensions/forms/base-fields/Selection/demos/#radio-buttons), I'm not able to:
- Tab into an option and select it using space/enter key
- Navigate between options using the arrow keys

I'm able to do all of the above in the [Radio component](https://eufemia.dnb.no/uilib/components/radio/#radio-group).

## To Reproduce

Visit the [demos of Field.Selection](https://eufemia.dnb.no/uilib/extensions/forms/base-fields/Selection/demos/#radio-buttons) and try to navigate using the keyboard.

## Expected behavior

It should work as in the [Radio component](https://eufemia.dnb.no/uilib/components/radio/#radio-group).
